sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ProcessBundleHandler.java

```diff
@@ -68,6 +68,7 @@
 
   // TODO: What should the initial set of URNs be?
   private static final String DATA_INPUT_URN = "urn:org.apache.beam:source:runner:0.1";
+  public static final String READ_URN = "urn:beam:transform:read:v1";
```

Comment:
<!--new_thread; commit:5300be61ccac7a006ec7f2b8f3824cddb4dc75e8; resolved:0-->
Can these URNs be imported from e.g. PTransformTranslation? We also do need to get them in a central place, I know, but anyhow in Java we have this available.

Reply:
There is a central place :-P https://github.com/apache/beam/blob/master/sdks/python/apache_beam/utils/urns.py 

Used (existing!) READ_URN from PTransformTranslation. The others are legacy/bundle specific, leaving for now.